<launch>
  <node pkg="ros2_evcam_driver" exec="event_histogram_publisher_node" name="event_histogram_publisher" output="screen">

    <param name="publish_period_ms" value="10" />
    <param name="histogram_window_ms" value="50" />
    <param name="count_cutoff" value="255" />
    <param name="downsample" value="false" />
    <param name="bins" value="1" />
  </node>

  <node pkg="event_histogram_accumulator" exec="histogram_accumulator_node" name="histogram_accumulator_node" output="screen">
    <param name="buffer_size" value="100" />
    <param name="output_bins" value="10" />
  </node>

  <node pkg="ros2_evcam_driver" exec="event_histogram_visualizer_node" name="event_histogram_visualizer_node" output="screen">
  </node>
  
</launch>
